mkdocs.yml

```diff
@@ -57,7 +57,6 @@ pages:
       Geography: entities/geography.md
       Product: entities/product.md
       Vessel: entities/vessel.md
-      Vessel Movement: entities/vessel_movement.md
```

Comment:
nit: should we also release a minor version of the previous semver (using python 3.7) with this removed? so that whoever doesn't want to move to a higher pythong version still has available a fully working version of the SDK

Reply:
or at least mark it as deprecated?

Reply:
This just removes a link to a non existent file in the docs. The link doesn't show because the file is non existent, so this change has no material effect on existing users.